sycl/source/handler.cpp

```diff
@@ -202,18 +202,26 @@ event handler::finalize() {
         std::move(MRequirements), std::move(MEvents), MCGType, MCodeLoc));
     break;
   case detail::CG::NONE:
-    throw runtime_error("Command group submitted without a kernel or a "
-                        "explicit memory operation.",
-                        PI_INVALID_OPERATION);
+    if (detail::pi::trace(detail::pi::TraceLevel::PI_TRACE_ALL)) {
+      std::cout << "The empty command group is supported by sycl2020"
```

Comment:
```suggestion
      std::cout << "WARNING: An empty command group is submitted."
```

Reply:
Do we have a configurable logger to deliver warnings from the runtime library to the user or we recommend to use `std::cout` for that?

Reply:
It's more like a logging information for SYCL library developers, so far we use `std::cout`.